test/libsolidity/semanticTests/builtinFunctions/keccak256_packed_complex_types.sol

```diff
@@ -10,5 +10,7 @@ contract C {
         hash3 = keccak256(abi.encodePacked(C(address(0x1234))));
     }
 }
+// ====
+// bytecodeFormat: legacy
```

Comment:
This should no longer need a separate EOF version after #15768. I see that quite a few tests I updated in that PR are still here. Do they not pass on EOF after all?

Reply:
It looks that something went totally wrong during rebasing on https://github.com/ethereum/solidity/pull/15768
Should be fixed now.